std/file.d

```diff
@@ -3142,7 +3142,7 @@ private struct DirIteratorImpl
 
         bool stepIn(string directory)
         {
-            auto h = cenforce(opendir(directory.tempCString()), directory);
+            auto h = cenforce(opendir(directory.length ? directory.tempCString() : "."), directory);
```

Comment:
> add a separate line to get C-string

I think it would have to be at least two lines (one to save `tempCString`'s result as its own type, otherwise the buffer's scope will end after the assignment statement)?